maths/print_multiplication_table.py

```diff
@@ -0,0 +1,26 @@
+def print_multiplication_table(number: int, number_of_terms: int) -> None:
```

Comment:
[Algorithmic functions should not print().](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md#what-is-an-algorithm)  So, instead of printing the results, this function should return a multi line string to the caller of the function.  The caller can then choose to print it, or put it in an email, or tweet it, etc.

```suggestion
def multiplication_table(number: int, number_of_terms: int) -> str:
```